src/main/resources/fetch.sql

```diff
@@ -0,0 +1,33 @@
+-- a. Retrieving all Homework records
+SELECT *
+FROM homework;
+SELECT name, description
+FROM homework;
+
+-- b. Retrieving all Lesson records, including Homework data
+SELECT lesson.name AS lesson_name, updated_at, homework.name AS homework_name, description
```

Comment:
```suggestion
SELECT lesson.name AS lesson_name, lesson.updated_at, homework.name AS homework_name, homework.description
```
looks more clear from my point of view